### ISSUE TITLE
Error in securityAssertion("Simulation from the model produced wrong dimension"

### ISSUE THREAD

**Issue by JEFFGAFNA16** [florianhartig/DHARMa#268]
Why do I get this error when running DHARMa. The model is okay but I seem to get the error message. At the tail end is the entire error message

gam_model <- gam(y ~ s(elevation))

library(DHARMa)
testDispersion(gam_model)

Error in securityAssertion("Simulation from the model produced wrong dimension",  : 
  Message from DHARMa: During the execution of a DHARMa function, some unexpected conditions occurred. Even if you didn't get an error, your results may not be reliable. Please check with the help if you use the functions as intended. If you think that the error is not on your side, I would be grateful if you could report the problem at https://github.com/florianhartig/DHARMa/issues 

 Context: Simulation from the model produced wrong dimension

**Comment by florianhartig** [comment]
Hi Jeff,

any chance you could provide some data so that we are able to run this ourselves?

Best,
Florian

**Comment by JEFFGAFNA16** [comment]
Dear Florian,
I tried this on another data set (mtcars) and it worked. I closed all other R studio that were opened and restarted my R studio. Surprisingly it is now working on my own data set. It works perfectly and I have my first plot. Am humbled by your quick response. Vielen Dank
![DHARMa plot](https://user-images.githubusercontent.com/24520144/111439367-976cac80-8705-11eb-96f6-3740bf41ea5d.png)
